src/cluster.cpp

```diff
@@ -421,6 +424,24 @@ CassError cass_cluster_set_use_randomized_contact_points(CassCluster* cluster,
   return CASS_OK;
 }
 
+CassError cass_cluster_set_constant_speculative_execution_policy(CassCluster* cluster,
+                                                                 cass_int64_t constant_delay_ms,
+                                                                 int max_speculative_executions) {
+  if (constant_delay_ms < 0 || max_speculative_executions < 0) {
+    return CASS_ERROR_LIB_BAD_PARAMS;
+  }
+  cluster->config().set_speculative_execution_policy(
+        new cass::ConstantSpeculativeExecutionPolicy(constant_delay_ms,
+                                                     max_speculative_executions));
+  return CASS_OK;
+}
+
+CassError cass_cluster_set_no_speculative_execution_policy(CassCluster* cluster) {
+  cluster->config().set_speculative_execution_policy(
+        new cass::NoSpeculativeExecutionPolicy());
```

Comment:
Agreed, not a huge deal here. That said, you *could* create a NoSpeculativeExecutionPlan data member in NoSpeculativeExecutionPolicy's constructor and return that one plan object anytime a user asks for a plan from this policy object. So, you won't have one NoSpeculativeExecutionPlan object in the whole application, but rather O(# NoSpeculativeExecutionPolicy objects), which is still better than O(# active requests). In fact, most of the time you'll only have one cluster object, so # of NoSpeculativeExecutionPolicy is often 1.